samples/app-hello-world/python/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-app-hello-world-python",
+    "source": "officeDev",
+    "title": "Microsoft Teams python Helloworld Sample",
+    "shortDescription": "Explore a comprehensive Microsoft Teams hello world sample app built with Node.js, demonstrating key features such as tabs, bots, and messaging extensions.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/app-hello-world/python",
+    "longDescription": [
+      "This Microsoft Teams Hello World sample app demonstrates the integration of tabs, bots, and messaging extensions using JavaScript, providing a comprehensive foundation for building interactive and engaging Teams applications. Experience the full capabilities of Teams by exploring features that enhance collaboration and productivity within the platform."
```

Comment:
Here its mentioned as JavaScript, this is a python sample update description

Reply:
fixed